daemons/compute/client-go/api/datamovement.pb.go

```diff
@@ -1,9 +1,24 @@
-// TODO: Copyright
```

Comment:
Like with the new c++ client, shouldn't we not check in these generated files?

Reply:
I prefer to leave the .go protobuf definitions in since we don't have the support of a Makefile to handle the generation prior `go build`

Reply:
Maybe that's something we do in the future.